examples/gocean/eg6/Makefile

```diff
@@ -45,10 +45,6 @@
 
 include ../../common.mk
 
-# Reset the default goal to be 'compile', i.e. to include compilation
-.DEFAULT_GOAL:=
-default: compile
```

Comment:
Is this deliberate: i.e. we want the default target to now be `transform`? If so, the `.PHONY` below needs updating too.

Reply:
As above ... transform is default, I've updated ``.PHONY``.